packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -78,14 +118,17 @@ class VMService {
   ///
   /// Requests made via the returns [VMService] time out after [requestTimeout]
   /// amount of time, which is [kDefaultRequestTimeout] by default.
+  /// If the [reloadSources] parameter is not null the 'reloadSources' service
```

Comment:
It's not clear from this comment what it means for the 'reloadSources' service to be registered.  Can you elaborate?